src/Illuminate/Contracts/Cache/Repository.php

```diff
@@ -3,8 +3,9 @@
 namespace Illuminate\Contracts\Cache;
 
 use Closure;
+use Psr\SimpleCache\CacheInterface;
 
-interface Repository
+interface Repository extends CacheInterface
```

Comment:
I feel that this should be `\Illuminate\Cache\Repository` that implements `Psr\SimpleCache\CacheInterface`